stripe-sms-receipt/assets/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This function shows you how to programmatically send a payment receipt URL to your customer via SMS using a <a href="https://stripe.com/docs/webhooks"> Stripe webhook event</a> as a trigger.</p>
+        <ol>
+          <li>Before you can test your app, you need to add a <a href="https://dashboard.stripe.com/webhooks"> Stripe Webhook</a> and set the URL to the URL of this page, but replace `/index.html` with `/send-sms-receipt`</li>
+          <li>Select "All Events" and save the webhook.</li>
+          <li>To test the app, initiate a payment from Stripe. You should receive an SMS receipt to the phone number on the payment.</li>
+        </ol>
+      </section>
```

Comment:
This app doesn't seem like you need to know the twilio phone number to execute it, so we need the `<!-- EDIT_CODE -->` here for just the buttons